src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj

```diff
@@ -33,12 +33,10 @@
     <Compile Remove="commands\utility\ShowCommand\ShowCommandParameterSetInfo.cs" />
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Online help tests for PowerShell Cmdlets.Validate 'get-help Update-List -Online'
```powershell
Expected strings to be the same, but they were different.
Expected length: 46
Actual length:   47
Strings differ at index 40.
Expected: '...nkID=11344...'
But was:  '...nkID=21093...'
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\HelpSystem.OnlineHelp.Tests.ps1: line 42
42:                 $actualURI | Should -Be $cmdlet.HelpURI
```